### ISSUE TITLE
Invalidation with multiple nodes

### ISSUE THREAD

**Issue by tillkruss** [swissspidy/performant-translations#68]
How does invalidation of changed MO files work if there is more than one web server?

Since only one machine will process the upgrader action, will the other machines still invalidate their cached translations?

**Comment by swissspidy** [comment]
Counter question: If you have multiple servers and update translations, how are they updated on the other servers? Surely they must be updated everywhere _somehow_.

I'd say how this works with this plugin very much depends on your individual setup. For example, if your `wp-content` directory is shared between all nodes, this is pretty much a non-issue, as then all servers use the same translations.

If you deploy updates manually, you might need to delete the `.php` files so that they can be regenerated.

Also worth noting that this becomes less of an issue once this plugin gets merged into core, as then translate.wordpress.org would automatically provide these `.php` files alongside the `.mo` files.

**Comment by tillkruss** [comment]
Yeah, unless the `WP_LAND_DIR` is shared between nodes, I don't think this plugin works with multi-node setups. Unless it's fully managed via Composer, etc and `DISALLOW_FILE_MODS` is set.

>Also worth noting that this becomes less of an issue once this plugin gets merged into core, as then translate.wordpress.org would automatically provide these .php files alongside the .mo files.

That makes sense.
